src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "country")
 public class Country implements Cloneable {
```

Comment:
The `Cloneable` interface is implemented here, but it doesn't seem to be used anywhere. Unless there is a specific requirement for the `Country` objects to be cloned, it would be better to remove this to avoid unnecessary complexity.